### PR TITLE
Hide specific aggregates for specific zones

### PR DESCRIPTION
**Pull request by madsnedergaard**
## Description

Following discussions prompted by #5780, we have decided to not show hourly data for zones where we don't actually have hourly data.

For Indonesia and Hong Kong specifically, we have an estimation model in place that takes a yearly value and averages it over lower periods all the way down to the hour. While this is useful on the API, we do not believe it portrays the two zones fairly on the map. Therefore we will hide data on zones that only support a higher aggregation than the one shown.

This means that if the zone only have daily data, we will hide it from the hourly view and only show it on the daily, monthly and yearly view.

As such the two particular zones should only show yearly data in the app.

Note that this unfortunately **does not** mean we can now support parsers with non-hourly data, as our infrastructure is still build around storing data on an hourly granularity.

### Preview

Screenshot 2023-09-19 at 15 25 35
Screenshot 2023-09-19 at 15 25 39

### Double check

- [x] I have run `pnpx prettier --write .` and `poetry run format` to format my changes.

### CODE REVIEW

**Review by VIKTORVAV99 — APPROVED** [review]
Just a small suggestion, but code wise it is looking good.

**Review by tonypls — APPROVED** [review]
LGTM, could be nice to hold off on merging until backend changes are deployed and we can test against the updated version 👍

File: web/public/locales/en.json [hunk 1]
```diff
@@ -66,6 +66,7 @@
     "aggregatedTooltip": "The data for this zone is aggregated based on historical hourly values",
     "helpUs": "Help us identify the problem by taking a look at the <a href=\"%s\" target=\"_blank\">runtime logs</a> or contact the data provider.",
     "noParserInfo": "We do not yet have any information about this area.<br/> Want to help fix this? You can contribute by <a href=\"%s\" target=\"_blank\">adding area data</a>.",
+    "aggregationDisabled": "Data is not available for the selected time period due to missing granular breakdown. Try a broader time period.",
```
[the remark this answers is not in the document]
Comment (madsnedergaard) [reply web/public/locales/en.json:69]: Yeah I was also struggling with the phrasing here 😄 
I don't think "lower granularity" is easily understood by most people, but I can't come up with a better alternative. Any ideas @tonypls?

[the remark this answers is not in the document]
Comment (VIKTORVAV99) [reply web/public/locales/en.json:69]: Maybe resolution would be a better word more people are familiar with?

[the remark this answers is not in the document]
Comment (tonypls) [reply web/public/locales/en.json:69]: Yeah I think granular for this use case is quite an Emaps context heavy term and it's not commonly used outside of scientific literature, particularly referring to time. Timeframe, time resolution are options but I think time period is probably the most common and the [wiktionary](https://en.wiktionary.org/wiki/time_period#:~:text=Noun,activity%20of%20condition%20being%20considered.) definitions matches the best with our use-case

**Pull request merged**